features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunBorderRouter.h

```diff
@@ -202,10 +202,10 @@ class WisunBorderRouter {
      * \param table_ptr Application allocated memory block where routing table is written.
      * \param table_len Length of the table allocated by application given as amount of entries.
      *
-     * \return MESH_ERROR_NONE on success.
-     * \return MESH_ERROR_UNKNOWN in case of failure.
+     * \returns 0 - x on success indicates number of route written to the table_ptr.
```

Comment:
should be "number of bytes"

Reply:
I guess it would be better to write:
`\returns 0 - x on success indicates number of entries written to the table_ptr.`

Reply:
It should be number of entries written so Debdeep suggestion is good